src/jest.config.ts

```diff
@@ -12,5 +12,6 @@ export default {
     ],
   ],
   testEnvironment: "node",
+  testMatch: ["**/*.test.ts"],
```

Comment:
Curious as to what prompted this?

Reply:
Ah - was really curious why the logging.test.ts file was running for the lambda unit tests.

Thought I'd pin down our filters for running tests whilst I was debugging.